lib/browser.js

```diff
@@ -0,0 +1,69 @@
+/*!
```

Comment:
This file name is a bit odd and could be mistaken for something else. Let's name it `middleware.js` unless you can think of something better.